### ISSUE TITLE
User profile page on smartphone shows JS alert "This page was unable to display a Google Maps element"

### ISSUE THREAD

**Issue by petervojtek** [PowerlineApp/powerline-mobile#80]
maybe due to:

```
Google Maps API warning: NoApiKeys https://developers.google.com/maps/documentation/javascript/error-messages#no-api-keys
util.js:220 Google Maps API warning: RetiredVersion https://developers.google.com/maps/documentation/javascript/error-messages#retired-version
util.js:220 Google Maps API warning: SensorNotRequired https://developers.google.com/maps/documentation/javascript/error-messages#sensor-not-required
```

**Comment by petervojtek** [comment]
this bug is present only on old (4.2) androids. on  android 5 and 6 the popup is not displayed, however the warning is displayed in console.

**Comment by petervojtek** [comment]
@jterps08 , the google maps API is used to suggest address/city in real-time when user is typing his address during Registration.

The use case currently works without the API key, but to make it bulletproof, we should generate some API key on behalf of Powerline.

https://developers.google.com/maps/documentation/javascript/get-api-key

**Comment by jterps08** [comment]
Got it - will send via Slack

**Comment by jterps08** [comment]
Sent

**Comment by petervojtek** [comment]
waiting to resolve how should we set the API key to codebase to protect the API key exposure..

**Comment by jterps08** [comment]
Is this closed @petervojtek ?

**Comment by petervojtek** [comment]
no @jterps08 , the issue still persists.

The point is:
I obtained the api keys from Austin, but I have not included them directly into source code as they will be visible to anyone who visits this repo. 

One way how it can be done is that we will build the apk/ipa on server via build process as was intended in #64, and here will be a step where the maps api keys will be bundled into the app just prior creating apk/ipa. However it seems #64 is not done at all.

In short, the lack of the keys in the app it is not harming the app functionality (now), but it may start later.

**Comment by jterps08** [comment]
Okay, I understand.

Can we do #64 or is that blocked by something / dependent on Austin or ?

**Comment by petervojtek** [comment]
@jterps08 , Austin had the aim to implement #64, not sure what is the state of the art of it since it is already open task for 3+ months..

**Comment by jterps08** [comment]
I'll make a comment over there...
